README.rst

```diff
@@ -145,20 +145,20 @@ This is not an oversight, it's a decision with the following rationale:
 1. Polling is "fast enough", particularly since PEP 471 introduced fast ``scandir``.
 
    With a reasonably large project like the TutorCruncher code base with 850 files and 300k lines
-   of code *watchgod* can scan the entire tree in ~24ms. With a scan interval of 400ms that's roughly
+   of code *watchgod* can scan the entire tree in ~24ms. The scan interval is 400ms which is roughly
```

Comment:
I don't think this is correct, the scan interval can be modified, 400ms is just an example.

Reply:
Reverted back to the original. Changed the start of the first sentence so that the 2 sentences don't start the same way.